package.json

```diff
@@ -13,9 +13,15 @@
   "license": "ISC",
   "dependencies": {
     "@cardano-ogmios/client": "^4.2.1",
+    "@emurgo/cardano-serialization-lib-nodejs": "^9.1.2",
+    "bech32": "^2.0.0",
     "big-integer": "^1.6.51",
     "bufferutil": "^4.0.5",
     "uniqid": "^5.4.0",
     "ws": "^8.4.0"
+  },
+  "devDependencies": {
```

Comment:
`purescript-language-server` and `pscid` should be provided by `easy-purescript-nix`